model/task_lifecycle.go

```diff
@@ -553,8 +556,17 @@ func doStepback(ctx context.Context, t *task.Task) error {
 		return nil
 	}
 
+	depth := 1
+	if t.StepbackInfo != nil {
```

Comment:
I'd be pro a lil function called t.GetStepbackDepth() so we could avoid needing this here.